app/assets/javascripts/discourse/app/models/pending-post.js

```diff
@@ -0,0 +1,18 @@
+import discourseComputed from "discourse-common/utils/decorators";
+import Category from "discourse/models/category";
+import RestModel from "discourse/models/rest";
+import { userPath } from "discourse/lib/url";
+
+const PendingPost = RestModel.extend({
+  @discourseComputed("category_id")
```

Comment:
This is a common pattern. I wonder if we should consider making it a mixin or helper.

Reply:
I had the same thought reading this.

Reply:
I can take a look :slightly_smiling_face:

Reply:
And done! I’ve kept it very simple for now.